wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -107,14 +112,23 @@ final class StartUIViewController: UIViewController {
         searchResultsViewController
     }
 
+    /// Whether there is a switch control for either listing/searching for users/people or apps/bots.
+    ///
+    /// The people/apps switch control will only be visible if
+    /// - apps/bots are not disabled for this build (restricted clients),
+    /// - the team's default protocol is Proteus the team has been using bots
+    /// - the team's default protocol is MLS and the `apps` feature flag is enabled.
     var showsGroupSelector: Bool {
-        guard DeveloperFlag.considerAppsFeatureFlag.isOn else {
-            return SearchGroup.all.count > 1 &&
-                userSession.selfUser.canSeeServices &&
-                userSession.defaultProtocol != .mls
+        guard SearchGroup.all.count > 1, userSession.selfUser.canSeeServices else { return false }
+
+        switch userSession.defaultProtocol {
+        case .mls:
+            return isAppsFeatureEnabled
+        case .proteus:
+            return areLegacyBotsAvailable
+        case .mixed:
+            return false
```

Comment:
question: can this really happen having mixed as defaultProtocol

Reply:
Probably not, but the enum has the mixed case so I need to handle it.